public/js/guests/controllers/guestListController.js

```diff
@@ -49,46 +50,45 @@ define([
 
         $scope.updateGuest = function (updatedGuest) {
             var alerts = validateGuest(updatedGuest),
-                id;
+                guestId;
 
             if (updatedGuest && alerts.length === 0) {
-                id = updatedGuest._id;
+                guestId = updatedGuest._id;
 
-                $http.put("/api/v1/guests/" + id, updatedGuest)
-                    .error(function (data) {
-                        $scope.alerts.push({msg: data, type: "danger"});
-                    })
-                    .success(function () {
+                GuestServices.update({ id:guestId }, updatedGuest).$promise.then(
+                    function success() {
                         angular.forEach($scope.guests, function (guest, index) {
-                            if (guest._id === id) {
+                            if (guest._id === guestId) {
                                 $scope.guests[index] = updatedGuest;
                             }
-                        });
-                    });
+                        })
+                    },
+                    function error(data) {
+                        $scope.alerts.push({msg: data, type: "danger"});
+                    }
+                );
             } else {
                 $scope.alerts = $scope.alerts.concat(alerts);
             }
         };
 
         $scope.deleteGuest = function (guest) {
-            var id = guest._id;
-            if (id) {
-                $http.delete("/api/v1/guests/" + id)
-                    .error(function (data) {
+            var guestId = guest._id;
+            if (guestId) {
+                GuestServices.delete({ id:guestId }).$promise.then(
+                    function success() {
+                    angular.forEach($scope.guests, function (guest, index) {
```

Comment:
your missing a closing bracket here. This line needs to be tabbed over and you'll see it